File: Client/Pikachu/PikachuLib/Models/Usermanagement/UserGroup.cs
using System;

namespace PikachuLib.Models.Usermanagement
{
    /// <summary>
    /// Repräsentation einer Benutzergruppe
    /// </summary>
    public class UserGroup
    {
        /// <summary>
        /// Id der Benutzergruppe
        /// </summary>
        public int Id { get; set; }

        /// <summary>
        /// Name der Benutzergruppe
        /// </summary>
        public string Name { get; set; }

        public UserGroup(int id, string name)
        {
            Id = id;
            Name = name;
        }

        public UserGroup(String name)
        {
            Name = name;
        }
    }
}

File: Server/relaxo/src/main/java/de/teamrocket/relaxo/models/usermanagement/UserGroup.java
package de.teamrocket.relaxo.models.usermanagement;

/**
 * Benutzergruppen werden durch diese Klasse repräsentiert.
 */
public class UserGroup {

    /**
     * Die ID der Gruppe.
     */
    private int id;

    /**
     * Die Bezeichnung der Gruppe.
     */
    private String name;

    public UserGroup() {
    }

    public String getName() {
        return this.name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getId() {
        return this.id;
    }

    public void setId(int id) {
        this.id = id;
    }

    @Override
    public String toString() {
        return "Gruppe: " + this.name + " | ID: " + this.id;
    }
}